README.md

```diff
@@ -32,23 +32,16 @@ Released     | Nightly
 This is the correct repo to file [issues](issues)
 
 
-
-## Important Note about the MSAL Preview
+## Support
 
 This library is suitable for use in a production environment.
-We provide the same production level support for this library as we do our current production libraries.
-During the preview we may make changes to the API, internal cache format, and other mechanisms of this library,
-which you will be required to take along with bug fixes or feature improvements.
-This may impact your application. For instance, a change to the cache format may impact your users,
-such as requiring them to sign in again. An API change may require you to update your code.
-When we provide the General Availability release we will require you to update to the General Availability version within six months,
-as applications written using a preview version of library may no longer work.
+
 
 ## Requirements
 
 * Windows 7 or greater
```

Comment:
Should we say Windows 8+

Reply:
I'm tempted to delete this altogheter. We support all operating systems via .net core, plus Android and iOS through Xamarin.

Reply:
Makes sense to me.

---
In reply to: [244970152](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/751#discussion_r244970152) [](ancestors = 244970152)